.gitignore

```diff
@@ -83,3 +84,19 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+
+gradle
```

Comment:
do you want this?

Reply:
Its where the gradle wrapper downloads the binaries.

Reply:
maybe add `/` then so it doesn't apply to a file named `gradle`.